profile/README.md

```diff
@@ -0,0 +1,33 @@
+<img align=right src="../logo.png"/>
+
+Welcome to the Scalameta GitHub organization! Scalameta's mission is to provide
+high quality and accessible tooling for Scala developers.
+
+| Our projects                                                                                                         |
+| -------------------------------------------------------------------------------------------------------------------- |
+| [Metals](https://scalameta.org/scalafmt): Scala language server                                                      |
+| [Scalafmt](https://scalameta.org/scalafmt): Scala code formatter                                                     |
+| [Scalafix](https://scalacenter.github.io/scalafix): Scala refactoring and linting tool                               |
+| [MDoc](https://scalameta.org/mdoc): Scala markdown documentation tool                                                |
+| [MUnit](https://scalameta.org/mdoc): Scala testing library                                                           |
+| [Trees](https://scalameta.org/docs/trees/guide.html): Scala tokenizer and parser                                     |
+| [SemanticDB](https://scalameta.org/docs/semanticdb/guide.html): Protobuf schema for modeling Scala symbols and types |
+
+| Our communication channels                                                                                        |
+| ----------------------------------------------------------------------------------------------------------------- |
+| [Twitter](https://twitter.com/scalameta): for announcements about new Scalameta releases and Scala developer tips |
+| [Discord](https://twitter.com/scalameta): to ask questions and discuss about Scalameta-related projects           |
+
+## Code of conduct
+
+The Scalameta organization is committed to provide a safe and welcoming
+environment for all people to learn, teach and contribute. We follow the
+[Scala code of conduct](https://www.scala-lang.org/conduct/). Don't hesitate to
+reach out to our organization admins to raise a problem or ask a question.
+
+| Organization admin                                  | Contact                |
+| --------------------------------------------------- | ---------------------- |
+| 🇺🇸 [Chris Kipp](https://github.com/ckipp01)         | ckipp@pm.me            |
```

Comment:
😬 Can you swap this?
```suggestion
| 🇺🇸 [Chris Kipp](https://github.com/ckipp01)         | open-source@chris-kipp.io            |
```
If not merge and I can update as well.

Reply:
Done.